azure_li_services/schema.py

```diff
@@ -0,0 +1,141 @@
+schema = {
+    'version': {
+        'required': True,
+        'type': 'date'
+    },
+    'instance_type': {
+        'required': False,
+        'type': 'string',
+        'nullable': False,
+        'regex': '^(LargeInstance|VeryLargeInstance)$'
+    },
+    'sku': {
+        'required': True,
+        'type': 'string'
+    },
+    'machine_constraints': {
+        'required': False,
+        'type': 'dict',
+        'schema': {
+            'min_cores': {
+                'required': False,
+                'type': 'number'
+            },
+            'min_memory': {
+                'required': False,
+                'type': 'string'
+            }
+        }
+    },
+    'networking': {
+        'required': True,
```

Comment:
I'm going back and fourth on this. Strictly speaking, if you don't want to set up a network, you shouldn't **have** to. But then it's crazy to not set up a network (at least for me, I have such extensive hatred of the browser-based console).

In terms of strict correctness, I'm unsure if the networking section is required. But we'll never NOT specify that, and we have no business need to not specify it. Given this, leave it alone, I'd say.

Reply:
As an aside, the networking section currently has no mechanism to set up bonding. This may be based strictly on `instance_type`, but I doubt it. But this is upcoming, so no need to worry about it now.

I love that there's unit tests - that helps to insure we don't break something along the way.

Reply:
yes unit tests give full freedom to do changes which is why I always write code with such tests.
So you are saying we should make networking optional ?